templates/base.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
Crossorigin Attribute:

Adding the crossorigin attribute to the stylesheet link provides an extra layer of security for loading external resources.